src/content/language/concurrency.md

```diff
@@ -428,6 +428,30 @@ objects, so they're subject to the same limitations.
 [`Pointer`]: {{site.dart-api}}/{{site.sdkInfo.channel}}/dart-ffi/Pointer-class.html
 [`UserTag`]: {{site.dart-api}}/{{site.sdkInfo.channel}}/dart-developer/UserTag-class.html
 
+#### Synchronous blocking communication between isolates
+
+There is a limit to the number of isolates that can run in parallel, or *synchronously*.
+This limit doesn't affect the standard *asynchronous* communication between isolates
+via messages in Dart. You can have hundreds of isolates running concurrently
+and making progress. The isolates are scheduled on the CPU in round-robin fashion,
+and yield to each other often.
+
+Isolates can only communicate *synchronously* outside of pure Dart,
+using C code via [FFI] to do so. 
+Attempts to perform synchronous communication between isolates in FFI calls
```

Comment:
I would add: "...between isolates" by doing synchronous blocking "in FFI calls..".